config/fcos/v1_6_exp/validate.go

```diff
@@ -15,13 +15,26 @@
 package v1_6_exp
 
 import (
+	"regexp"
+
 	"github.com/coreos/butane/config/common"
 	"github.com/coreos/ignition/v2/config/util"
 
 	"github.com/coreos/vcontext/path"
 	"github.com/coreos/vcontext/report"
 )
 
+var regex = regexp.MustCompile(`^/(etc|var)/.*`)
+
+func (conf Config) Validate(c path.ContextPath) (r report.Report) {
+	for _, fs := range conf.Storage.Filesystems {
+		if util.NilOrEmpty(fs.Path) || (!regex.MatchString(*fs.Path)) {
```

Comment:
It's legal not to specify the `path`, if you want to format a filesystem but not mount it.

Reply:
This still needs to be fixed.  The code currently fails if the path is unspecified.